src/FluentValidation/DefaultValidatorExtensions.cs

```diff
@@ -1057,5 +1057,58 @@ public static class DefaultValidatorExtensions {
 		public static IRuleBuilderInitial<T, TProperty> CustomAsync<T, TProperty>(this IRuleBuilder<T, TProperty> ruleBuilder, Func<TProperty, CustomContext, CancellationToken, Task> action) {
 			return (IRuleBuilderInitial<T, TProperty>) ruleBuilder.SetValidator(new CustomValidator<TProperty>(action));
 		}
+
+		/// <summary>
+		/// Defines a boolean validator on the current rule builder.
+		/// Validation will fail if the property is true.

```

Comment:
This is the wrong way around. For the `True` method, validation will fail if the property is `false`.